README.md

```diff
@@ -1,123 +1,48 @@
 <h1 align=center>Pinwheel Astro</h1>
```

Comment:
Let's change this title too.
Give credits in the readme and links to original forked repo.

Reply:
Understood